docs/prob/api/unauthorized.md

```diff
@@ -0,0 +1,28 @@
+# Problem: Unauthorized
+
+<!-- panels:start -->
+
+<!-- div:right-panel -->
+
+- Title: `Unauthorized.`
+- Type: `/prob/api/unauthorized`
+
+<!-- div:left-panel -->
+
+Failed to authenticate, or insufficient permissions.
+
+<!-- panels:end -->
+
+## Possible causes
+
+- Invalid token.
+- Invalid username and/or password.
+- You might have insufficient permissions to perform the requested operation.
```

Comment:
would be nice to add a note that this might be insufficient in regard to Wharf's user base or the remote provider